python-avd/tox.ini

```diff
@@ -43,3 +43,4 @@ dependency_groups =
 commands =
     coverage report --rcfile=pyproject.toml
     coverage xml --rcfile=pyproject.toml
+    coverage html --rcfile=pyproject.toml
```

Comment:
why?

Reply:
hmm to have a local html view of coerage - pushed by mistake.. :)

Reply:
```suggestion
```